pkg/collectors/policyreport.go

```diff
@@ -95,10 +95,15 @@ func createPolicyReportListWatchWithClient(client dynamic.Interface, ns string)
 
 func getReports(clusterID string, pr *v1alpha2.PolicyReport) [][]string {
 	var metrics [][]string
+	var severity string
+
 	if clusterID == "" {
 		return metrics
 	}
-	category, policy, result, severity := "", "", "", ""
+
+	category := "CM Configuration Management"
+	result := "fail"
```

Comment:
Alternatively, maybe we keep the empty string and just remove the empty category condition when appending? What would be the consequence there?

I'm also curious about this loop because it seems when it hits a `reportResult` with a category, it gets populated and then would have a `category` for every `reportResult` after that. It seems this should be set as a loop variable rather than outside of the loop?

Reply:
Okay, another thought: we scrap the `category` and `result` variables entirely and use them directly in the `append()`?.

Reply:
> Okay, another thought: we scrap the `category` and `result` variables entirely and use them directly in the `append()`?.

^^^ (Since I've brain dumped into this issue 😄): Unless there's a reason a category is required, I think leaving the category empty makes the most sense since it'd match what the customer has on the cluster.